### PR TITLE
Milestone: Hooks

### PR DESCRIPTION
**Pull request by AlexRS90**
## For this milestone I:
- Deleted class properties at Calculator.js ✂️ 
- Built a new arrow function 🏹 
- Added hooks 🏴‍☠️

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,59 +1,51 @@
...
-  }
+const initial = {
+  total: '0',
+  next: '',
+  operation: '',
+};
 
-  handleClick = (obj, btnValue) => {
-    const newObj = calculate(obj, btnValue);
+const Calculator = () => {
```
Comment (GzimAsani) [inline src/components/Calculator.js:10]: - [ ] When I use the modulus with 0 it throws an error, please handle this by displaying a message to the user that they cannot use modulus with 0

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @AlexRS90                              
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (AlexRS90) [reply src/components/Calculator.js:10]: Hi @GzimAsani thanks for your suggestion, can you explain to me, what do you mean with 0?

Comment (GzimAsani) [reply src/components/Calculator.js:10]: When I do for example 8 % 0 or anything with 0 it throws an error. Sry for the late reply

Comment (AlexRS90) [reply src/components/Calculator.js:10]: Thank you for clarifying it, yes that's true I didn't notice yet! The error is only with % 0, I'll fix it!!

**Review by Lameck1 — APPROVED** [review]
Hi, @AlexRS90,

You've done well addressing the concerns from the previous review 👏🏼.

The tasks for this milestone are now successfully implemented.
All the best with the next milestone.

You are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AlexRS90** [comment]
Thank you @Lameck1

**Pull request merged**